packages/lib-utils/src/components/list-view/ListView.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import '@patternfly/react-core/dist/styles/base.css';
```

Comment:
> If you have global imported styles, create a file called [.storybook/preview.js](https://storybook.js.org/docs/react/configure/overview#configure-story-rendering) and import the styles there. They will be added by Storybook automatically for all stories.

I think we can import ^^ from the preview config file, so we don't have to repeat it in every stories file.